### PR TITLE
Rename dataRepo and org in the local storage

### LINKED ISSUE #106: Use different names for dataRepo and org in local storage
Currently, WATcher stores the repo name in `dataRepo` and `org` fields in the local storage. This interferes with CATcher, since both applications are using the same fields and they are hosted on the same origin. There could be potential issues with this if a person uses CATcher and WATcher at the same time.

Let's rename the fields to something else, like `WATcher:dataRepo` and `WATcher:org`

### PR DESCRIPTION
**Pull request by wweqg**
### Summary:

Fixes #106 

#### Type of change:

- 🎨 Code Refactoring

### Changes Made:

Rename dataRepo and org in the local storage to WATcher:dataRepo and WATcher:org

### Proposed Commit Message:

```
Rename dataRepo and org to WATcher:dataRepo and WATcher:org

This commit changes the names of the data repository and the organization
to include the 'WATcher' prefix. The 'dataRepo' is now 'WATcher:dataRepo' 
and the organization name is changed to 'WATcher:org'. 

```

<details><summary>
<h3>Checklist:</h3>
</summary>

- [x] I have tested my changes thoroughly.
- [ ] I have created tests for any new code files created in this PR or provided a link to a issue/PR that addresses this.
- [ ] I have added or modified code comments to improve code readability where necessary.
- [ ] I have updated the project's documentation as necessary.

</details>

### CODE REVIEW

File: src/app/core/services/phase.service.ts [hunk 1]
```diff
@@ -125,9 +125,9 @@ export class PhaseService {
    * Retrieves the repository url from local storage and sets to current repository.
    */
   async initializeCurrentRepository() {
-    const org = window.localStorage.getItem('org');
-    const repoName = window.localStorage.getItem('dataRepo');
-    this.logger.info(`PhaseService: received initial org (${org}) and initial name (${repoName})`);
```
Comment (chunweii) [inline src/app/core/services/phase.service.ts:130]: Let's not change the logging message in the PR

**Review by chunweii — CHANGES_REQUESTED** [review]
Thank you for your PR @wweqg. It would be better to extract all the magic string literals into a constant instead

File: src/app/core/services/phase.service.ts (hunk 1 above)
Comment (wweqg) [reply src/app/core/services/phase.service.ts:130]: Okay. I have updated it accordingly.

**Comment by wweqg** [comment]
> Thank you for your PR @wweqg. It would be better to extract all the magic string literals into a constant instead

Thank @chunweii for the comment. I have extracted the string literals into constants.

**Review by chunweii — APPROVED** [review]

**Comment by chunweii** [comment]
Please follow the git commit message conventions in https://se-education.org/guides/conventions/git.html

In particular, please have a proper subject, and a body

**Comment by wweqg** [comment]
> Please follow the git commit message conventions in https://se-education.org/guides/conventions/git.html
> 
> In particular, please have a proper subject, and a body

Thanks for the reminder. I have updated the proposed commit message

**Pull request merged**